cookbooks/hardware/templates/default/ilo-defaults.xml.erb

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<RIBCL VERSION="2.0">
+  <LOGIN USER_LOGIN="admin" PASSWORD="password">
+    <SERVER_INFO mode="write">
+      <!-- Set Server Name -->
+      <SERVER_NAME VALUE="<%= node[:fqdn] %>"/>
+      <!-- 1 Operating system control mode -->
+      <!-- 2 HP Static Low Power mode -->
+      <!-- 3 HP Dynamic Power Savings mode -->
+      <!-- 4 HP Static High Performance mode -->
+      <SET_HOST_POWER_SAVER HOST_POWER_SAVER="3"/>
+      <!-- Set Auto Power ON -->
+      <!-- Set Auto Power after random delay -->
+      <!-- Both are required -->
+      <SERVER_AUTO_PWR VALUE="ON"/>
```

Comment:
What exactly does this mean? Presumably it won't power back on if you power it down so is this just after a power restoration? What if the machine was powered off before the power loss?

Reply:
On power restore automatically power on the system. Opens are ON, OFF, LAST.

Reply:
> What if the machine was powered off before the power loss?
I think the system will then power ON again after the power loss. I have tended to always set to ON.

Reply:
I usually go for LAST on PCs but it's not reliable then fine - it's true our machines do normally mysteriously power up after a power loss even if they were off before.